README.md

```diff
@@ -125,6 +102,70 @@ With respect to the `tags` configuration option, the tags that will be added are
+* `debug` - Should we output lots of debug info? :)
+* `hostname` - The hostname to be used with each metric sent. Defaults to `os.Hostname()`
+* `interval` - How often to flush. Something like 10s seems good.
+* `key` - Your Datadog API key
+* `percentiles` - The percentiles to generate from our timers and histograms. Specified as array of float64s
+* `udp_address` - The address on which to listen for metrics. Probably `:8126` so as not to interfere with normal DogStatsD.
+* `http_address` - The address to serve HTTP healthchecks and other endpoints. This can be a simple ip:port combination like `127.0.0.1:8127`. If you're under einhorn, you probably want `einhorn@0`.
+* `forward_address` - The address of an upstream Veneur to forward metrics to. See below.
+* `num_workers` - The number of worker goroutines to start.
+* `num_readers` - The number of reader goroutines to start. Veneur supports SO_REUSEPORT on Linux to scale to multiple readers. On other platforms, this should always be 1; other values will probably cause errors at startup. See below.
+* `read_buffer_size_bytes` - The size of the receive buffer for the UDP socket. Defaults to 2MB, as having a lot of buffer prevents packet drops during flush!
+* `stats_address` - The address to send internally generated metrics. Probably `127.0.0.1:8125` to send to a local DogStatsD
+* `tags` - Tags to add to every metric that is sent to Veneur. Expects an array of strings!
+
+# Monitoring
+
+Here are the important things to monitor with Veneur:
+
+# At Local Node
+
+When running as a local instance, you will be primarily concerned
+* Datadog's `system.net.udp.in_errors` should be monitored, as drops imply missed metrics. Even better, monitor it as a percentage of `system.net.udp.in_datagrams`.
+
+## Forwarding
+
+If you are forwarding metrics to central Veneur, you'll want to monitor these:
+* `veneur.forward.error_total` and it's `cause` tag. This should pretty much never happen and definitely not be sustained.
+* `veneur.forward.duration_ns` and `veneur.forward.duration_ns.count`. These metrics track the per-host time spent performing a forward. The time should be minimal!
+
+
+# At Global Node
+
+When forwarding you'll want to also monitor the global nodes you're using for aggregation:
+* `veneur.import.request_error_total` and it's `cause` tag. This should pretty much never happen and definitely not be sustained.
+* `veneur.import.response_duration_ns` and `veneur.import.response_duration_ns.count` to monitor duration and number of received forwards. This should not fail and not take very long. How long it takes will depend on how many metrics you're forwarding.
+
+# Metrics
+
+Veneur assumes you have a running DogStatsD on the localhost and emits metrics to it's default port of 8125. Those metrics are:
```

Comment:
should probably refer to the `stats_address` config option here? since that determines where it sends these